### PR TITLE
[Containerapp] `az containerapp env create`: Deprecate unused argument `--docker-bridge-cidr`

### PR DESCRIPTION
**Pull request by Greedygre**
**Related command**

Option '--docker-bridge-cidr' has been deprecated and will be removed in a future release.
image

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Containerapp

**Review by Juliehzl — APPROVED** [review]

**Review by LaylaLiu-gmail — APPROVED** [review]

**Comment by yanzhudd** [comment]
please rerun relative tests to verify this PR won't bring any unexpected change.

**Comment by Greedygre** [comment]
> please rerun relative tests to verify this PR won't bring any unexpected change.

Hi @yanzhudd 

Rerun relative tests and add one test for the deprecated argument.

File: src/azure-cli/azure/cli/command_modules/containerapp/_params.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp env', arg_group='Virtual Network') as c:
         c.argument('infrastructure_subnet_resource_id', options_list=['--infrastructure-subnet-resource-id', '-s'], help='Resource ID of a subnet for infrastructure components and user app containers.')
-        c.argument('docker_bridge_cidr', options_list=['--docker-bridge-cidr'], help='CIDR notation IP range assigned to the Docker bridge. It must not overlap with any Subnet IP ranges or the IP range defined in Platform Reserved CIDR, if defined')
+        c.argument('docker_bridge_cidr', options_list=['--docker-bridge-cidr'], help='CIDR notation IP range assigned to the Docker bridge. It must not overlap with any Subnet IP ranges or the IP range defined in Platform Reserved CIDR, if defined', deprecate_info=c.deprecate(target='--docker-bridge-cidr', hide=True))
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/containerapp/_params.py:164]: add target version?

Comment (Greedygre) [reply src/azure-cli/azure/cli/command_modules/containerapp/_params.py:164]: The latest version include this change is `2.65.0`, I will use `2.68.0` as the target expiration version:
image

**Review by Juliehzl — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**